source/apiVolontaria/apiVolontaria/tests/tests_view_Users.py

```diff
@@ -450,4 +534,4 @@ def test_list_users_without_permissions(self):
         content = {"detail": "You are not authorized to list users."}
         self.assertEqual(json.loads(response.content), content)
 
-        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
```

Comment:
Please, let a newline at the end of the file for PEP8 verification

Reply:
done

Reply:
done